hack/lib/serverless.bash

```diff
@@ -100,35 +100,32 @@ function find_install_plan {
   echo ""
 }
 
-# Deploys the version of KnativeServing from 1.3.0 release. The future releases should ensure compatibility with
-# this custom resource and accept it.
-function deploy_knativeserving_v1alpha1_1.3.0 {
+function deploy_knativeserving_cr {
   logger.info 'Deploy Knative Serving'
 
   # Wait for the CRD to appear
   timeout 900 "[[ \$(oc get crd | grep -c knativeservings) -eq 0 ]]" || return 6
 
-  local rootdir
-  rootdir="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
-
-  # Install Knative Serving
-  oc apply -n "${SERVING_NAMESPACE}" -f "${rootdir}/test/v1alpha1/resources/serving.knative.dev_v1alpha1_knativeserving_cr_1.3.0.yaml" || return $?
```

Comment:
We still provide an example here: https://github.com/openshift-knative/serverless-operator/pull/157/files#diff-b466a37bd25835bf7c89df6eef60bb62R5 and we allow the user to specify various attributes. It would be good to create a file named `operator.knative.dev_v1alpha1_knativeserving_cr_1.6.0.yaml` with those various attributes (maybe same as the example?) and make sure we use it with the future releases. This way we will be able to verify that the previous config (with all the attributes) can still be used to install the latest Serverless.
What I see below is a minimal version of the config but it doesn't help check backwards compatibility. And this minimal config is already used in the Golang tests [here](https://github.com/openshift-knative/serverless-operator/blob/master/test/v1alpha1/knativeserving.go#L13) so no need to duplicate here, IMO.